### ISSUE TITLE
Saving the project failed because of a Server Error

### ISSUE THREAD

**Issue by jake-low** [osmus/tasking-manager#280]
Hello,

I'm attempting to create my first task in the OSMUS Tasking Manager and I'm running into an issue which I'm unable to figure out. It's manifesting as what appears to be a server error but I'm not sure if that's the root cause. Here's what I'm seeing:

I created a new task, uploading a GeoJSON boundary to define the task area. The task is now listed on my personal dashboard in draft state, as expected.

I then tried to edit the task to add the required fields and publish it. The fields that were marked as required which I filled out were:
- Description > Name of the project (already filled in during creation)
- Description > Short description
- Description > Description
- Instructions > Detailed instructions
- Metadata > Types of mapping
- Metadata > Organization (already filled in during creation)

Afterwards I hit the "Save" button in the sidebar, this error appeared: "Saving the project failed because of a Server Error. Please try again later or contact the administrator if problem persists."

<img width="500" alt="image" src="https://github.com/osmus/tasking-manager/assets/5893857/5899cd7b-0ae8-4d90-a8d1-bc5a1e518680">

I checked the browser devtools and saw that the underlying error was from `PATCH /api/v2/projects/585`. The response body was:
```
{"Error": "Unable to update project", "SubCode": "InternalServerError"}
```

I tried this in Chrome, Safari and Firefox and saw the same failure in all cases.

Here's the full curl-encoded version of the request that triggered that response (with my auth token redacted). There should hopefully be corresponding server log entries for my repeated attempts to save the form in the window from about 4:30pm to 5pm PST today (2024-05-13T23:30:00Z to 2024-05-14T00:00:00Z). 

```
curl 'https://tasks-backend.openstreetmap.us/api/v2/projects/585/' \
  -X 'PATCH' \
  -H 'authority: tasks-backend.openstreetmap.us' \
  -H 'accept: */*' \
  -H 'accept-language: en' \
  -H 'authorization: Token <REDACTED>' \
  -H 'content-type: application/json' \
  -H 'origin: https://tasks.openstreetmap.us' \
  -H 'referer: https://tasks.openstreetmap.us/' \
  -H 'sec-ch-ua: "Not?A_Brand";v="8", "Chromium";v="108", "Google Chrome";v="108"' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'sec-ch-ua-platform: "macOS"' \
  -H 'sec-fetch-dest: empty' \
  -H 'sec-fetch-mode: cors' \
  -H 'sec-fetch-site: same-site' \
  -H 'user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36' \
  --data-raw $'{"projectId":585,"database":"OSM","status":"DRAFT","projectPriority":"MEDIUM","areaOfInterest":{"type":"MultiPolygon","coordinates":[[[[-108.808764,42.521381],[-109.295413,42.529663],[-109.29297,42.619671],[-109.414848,42.621433],[-109.412578,42.711459],[-109.534636,42.71309],[-109.532532,42.803105],[-109.654776,42.804611],[-109.652839,42.894637],[-109.77527,42.896018],[-109.773498,42.986044],[-109.896116,42.987291],[-109.887976,43.437541],[-109.764442,43.43628],[-109.762594,43.526296],[-109.515036,43.52335],[-109.519429,43.343335],[-109.396106,43.341663],[-109.400813,43.161651],[-109.277867,43.159858],[-109.280375,43.069842],[-109.157622,43.067932],[-109.160297,42.977927],[-109.037738,42.975889],[-109.04343,42.795908],[-108.921231,42.793741],[-108.924227,42.703747],[-108.802218,42.701459],[-108.808546,42.521381],[-108.808764,42.521381]]]]},"aoiBBOX":[-109.896116,42.521381,-108.802218,43.526296],"tasks":{"type":"FeatureCollection","features":[{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-110.039062,42.940339],[-109.863281,42.940339],[-109.863281,43.068888],[-110.039062,43.068888],[-110.039062,42.940339]]]]},"properties":{"taskId":1,"taskX":398,"taskY":1295,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-110.039062,43.068888],[-109.863281,43.068888],[-109.863281,43.197167],[-110.039062,43.197167],[-110.039062,43.068888]]]]},"properties":{"taskId":2,"taskX":398,"taskY":1296,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-110.039062,43.197167],[-109.863281,43.197167],[-109.863281,43.325178],[-110.039062,43.325178],[-110.039062,43.197167]]]]},"properties":{"taskId":3,"taskX":398,"taskY":1297,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-110.039062,43.325178],[-109.863281,43.325178],[-109.863281,43.452919],[-110.039062,43.452919],[-110.039062,43.325178]]]]},"properties":{"taskId":4,"taskX":398,"taskY":1298,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.863281,42.811522],[-109.6875,42.811522],[-109.6875,42.940339],[-109.863281,42.940339],[-109.863281,42.811522]]]]},"properties":{"taskId":5,"taskX":399,"taskY":1294,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.863281,42.940339],[-109.6875,42.940339],[-109.6875,43.068888],[-109.863281,43.068888],[-109.863281,42.940339]]]]},"properties":{"taskId":6,"taskX":399,"taskY":1295,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.863281,43.068888],[-109.6875,43.068888],[-109.6875,43.197167],[-109.863281,43.197167],[-109.863281,43.068888]]]]},"properties":{"taskId":7,"taskX":399,"taskY":1296,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.863281,43.197167],[-109.6875,43.197167],[-109.6875,43.325178],[-109.863281,43.325178],[-109.863281,43.197167]]]]},"properties":{"taskId":8,"taskX":399,"taskY":1297,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.863281,43.325178],[-109.6875,43.325178],[-109.6875,43.452919],[-109.863281,43.452919],[-109.863281,43.325178]]]]},"properties":{"taskId":9,"taskX":399,"taskY":1298,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.863281,43.452919],[-109.6875,43.452919],[-109.6875,43.580391],[-109.863281,43.580391],[-109.863281,43.452919]]]]},"properties":{"taskId":10,"taskX":399,"taskY":1299,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.6875,42.682435],[-109.511719,42.682435],[-109.511719,42.811522],[-109.6875,42.811522],[-109.6875,42.682435]]]]},"properties":{"taskId":11,"taskX":400,"taskY":1293,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.6875,42.811522],[-109.511719,42.811522],[-109.511719,42.940339],[-109.6875,42.940339],[-109.6875,42.811522]]]]},"properties":{"taskId":12,"taskX":400,"taskY":1294,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.6875,42.940339],[-109.511719,42.940339],[-109.511719,43.068888],[-109.6875,43.068888],[-109.6875,42.940339]]]]},"properties":{"taskId":13,"taskX":400,"taskY":1295,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.6875,43.068888],[-109.511719,43.068888],[-109.511719,43.197167],[-109.6875,43.197167],[-109.6875,43.068888]]]]},"properties":{"taskId":14,"taskX":400,"taskY":1296,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.6875,43.197167],[-109.511719,43.197167],[-109.511719,43.325178],[-109.6875,43.325178],[-109.6875,43.197167]]]]},"properties":{"taskId":15,"taskX":400,"taskY":1297,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.6875,43.325178],[-109.511719,43.325178],[-109.511719,43.452919],[-109.6875,43.452919],[-109.6875,43.325178]]]]},"properties":{"taskId":16,"taskX":400,"taskY":1298,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.6875,43.452919],[-109.511719,43.452919],[-109.511719,43.580391],[-109.6875,43.580391],[-109.6875,43.452919]]]]},"properties":{"taskId":17,"taskX":400,"taskY":1299,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.511719,42.55308],[-109.335937,42.55308],[-109.335937,42.682435],[-109.511719,42.682435],[-109.511719,42.55308]]]]},"properties":{"taskId":18,"taskX":401,"taskY":1292,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.511719,42.682435],[-109.335937,42.682435],[-109.335937,42.811522],[-109.511719,42.811522],[-109.511719,42.682435]]]]},"properties":{"taskId":19,"taskX":401,"taskY":1293,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.511719,42.811522],[-109.335937,42.811522],[-109.335937,42.940339],[-109.511719,42.940339],[-109.511719,42.811522]]]]},"properties":{"taskId":20,"taskX":401,"taskY":1294,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.511719,42.940339],[-109.335937,42.940339],[-109.335937,43.068888],[-109.511719,43.068888],[-109.511719,42.940339]]]]},"properties":{"taskId":21,"taskX":401,"taskY":1295,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.511719,43.068888],[-109.335937,43.068888],[-109.335937,43.197167],[-109.511719,43.197167],[-109.511719,43.068888]]]]},"properties":{"taskId":22,"taskX":401,"taskY":1296,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.511719,43.197167],[-109.335937,43.197167],[-109.335937,43.325178],[-109.511719,43.325178],[-109.511719,43.197167]]]]},"properties":{"taskId":23,"taskX":401,"taskY":1297,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.511719,43.325178],[-109.335937,43.325178],[-109.335937,43.452919],[-109.511719,43.452919],[-109.511719,43.325178]]]]},"properties":{"taskId":24,"taskX":401,"taskY":1298,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.335937,42.423457],[-109.160156,42.423457],[-109.160156,42.55308],[-109.335937,42.55308],[-109.335937,42.423457]]]]},"properties":{"taskId":25,"taskX":402,"taskY":1291,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.335937,42.55308],[-109.160156,42.55308],[-109.160156,42.682435],[-109.335937,42.682435],[-109.335937,42.55308]]]]},"properties":{"taskId":26,"taskX":402,"taskY":1292,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.335937,42.682435],[-109.160156,42.682435],[-109.160156,42.811522],[-109.335937,42.811522],[-109.335937,42.682435]]]]},"properties":{"taskId":27,"taskX":402,"taskY":1293,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.335937,42.811522],[-109.160156,42.811522],[-109.160156,42.940339],[-109.335937,42.940339],[-109.335937,42.811522]]]]},"properties":{"taskId":28,"taskX":402,"taskY":1294,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.335937,42.940339],[-109.160156,42.940339],[-109.160156,43.068888],[-109.335937,43.068888],[-109.335937,42.940339]]]]},"properties":{"taskId":29,"taskX":402,"taskY":1295,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.335937,43.068888],[-109.160156,43.068888],[-109.160156,43.197167],[-109.335937,43.197167],[-109.335937,43.068888]]]]},"properties":{"taskId":30,"taskX":402,"taskY":1296,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.160156,42.423457],[-108.984375,42.423457],[-108.984375,42.55308],[-109.160156,42.55308],[-109.160156,42.423457]]]]},"properties":{"taskId":31,"taskX":403,"taskY":1291,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.160156,42.55308],[-108.984375,42.55308],[-108.984375,42.682435],[-109.160156,42.682435],[-109.160156,42.55308]]]]},"properties":{"taskId":32,"taskX":403,"taskY":1292,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.160156,42.682435],[-108.984375,42.682435],[-108.984375,42.811522],[-109.160156,42.811522],[-109.160156,42.682435]]]]},"properties":{"taskId":33,"taskX":403,"taskY":1293,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.160156,42.811522],[-108.984375,42.811522],[-108.984375,42.940339],[-109.160156,42.940339],[-109.160156,42.811522]]]]},"properties":{"taskId":34,"taskX":403,"taskY":1294,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-109.160156,42.940339],[-108.984375,42.940339],[-108.984375,43.068888],[-109.160156,43.068888],[-109.160156,42.940339]]]]},"properties":{"taskId":35,"taskX":403,"taskY":1295,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-108.984375,42.423457],[-108.808594,42.423457],[-108.808594,42.55308],[-108.984375,42.55308],[-108.984375,42.423457]]]]},"properties":{"taskId":36,"taskX":404,"taskY":1291,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-108.984375,42.55308],[-108.808594,42.55308],[-108.808594,42.682435],[-108.984375,42.682435],[-108.984375,42.55308]]]]},"properties":{"taskId":37,"taskX":404,"taskY":1292,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-108.984375,42.682435],[-108.808594,42.682435],[-108.808594,42.811522],[-108.984375,42.811522],[-108.984375,42.682435]]]]},"properties":{"taskId":38,"taskX":404,"taskY":1293,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-108.808594,42.423457],[-108.632812,42.423457],[-108.632812,42.55308],[-108.808594,42.55308],[-108.808594,42.423457]]]]},"properties":{"taskId":39,"taskX":405,"taskY":1291,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-108.808594,42.55308],[-108.632812,42.55308],[-108.632812,42.682435],[-108.808594,42.682435],[-108.808594,42.55308]]]]},"properties":{"taskId":40,"taskX":405,"taskY":1292,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}},{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-108.808594,42.682435],[-108.632812,42.682435],[-108.632812,42.811522],[-108.808594,42.811522],[-108.808594,42.682435]]]]},"properties":{"taskId":41,"taskX":405,"taskY":1293,"taskZoom":11,"taskIsSquare":true,"taskStatus":"READY","lockedBy":null,"mappedBy":null}}]},"defaultLocale":"en","projectInfo":{"locale":"en","name":"Wind River Range lake polygon improvement","shortDescription":"","description":"","instructions":"","perTaskInstructions":""},"projectInfoLocales":[{"locale":"en","name":"Wind River Range, WY lake polygons","shortDescription":"Use the provided lake polygons dataset (generated from 1m LiDAR DEM) to improve hydrography data in the Wind River Range\\n","description":"The Wind River Range is a mountain range in western Wyoming. Much of it is designated wilderness and offers spectacular hiking and climbing. There are about 2600 lakes in the Wind River Range in OSM. Of these, about half were last touched in 2016 or earlier, when available imagery in the region was poor, and as a result a lot of these lakes are mapped very coarsely (often hundreds of meters between nodes on the perimeter way).\\n\\nToday, much better imagery is available, including medium-resolution NAIP and USGS imagery as well as high resolution imagery from Bing and ESRI which appears to be very well orthorectified.\\n\\nThere is also 1m LiDAR-derived digital elevation model (DEM) data available for the whole region from USGS, which is in the public domain. **Using this data, I was able to generate very accurate polygons for thousands of lakes in the range.**\\n\\nThe goal of this task is to use the generated lake polygons to improve OSM, by reviewing them one at a time and (if appropriate) incorporate them into the map, replacing existing low-precision lake geometry.\\n\\n**Use of the JOSM editor is recommended.** I don\'t believe that the proposed workflow is currently possible in iD.","instructions":"Before starting to map this task, please read the [community forum post](https://community.openstreetmap.org/t/improving-lake-polygons-in-wind-river-range-wy/113089) describing this project. Pay attention especially to the example images showing data errors you should be watchful for when reviewing the generated lake polygons.\\n\\n1. Download the [generated lake polygons GeoJSON file](https://gist.github.com/jake-low/3293df9d331da225c94b00a32384e2dd/)\\n2. Choose a task unit in the tasking manager and open it in JOSM.\\n3. Open the GeoJSON file in JOSM as a separate data layer.\\n4. Enable the recommended satellite imagery layers: Bing, ESRI Clarity, National Agriculture Imagery Program, and USGS Imagery. All are fairly well orthorectified (USGS has some issues in steep terrain). Bing and ESRI are the highest resolution. NAIP has the least snow cover.\\n5. Examine each generated lake polygon from the GeoJSON file within the bounds of the task unit (ignore lakes where more than 50% of the polygon lies outside the bounds of your chosen task area).\\n   - If the polygon represents a real lake (visible in the imagery layers), and that lake is either missing from OSM or is mapped at lower precision than the generated polygon, select the generated polygon and copy it (Cmd/Ctrl + C). Then activate the OSM data layer and paste it **at the source position** (Cmd/Ctrl + Opt/Alt + V). Some polygons in the generated dataset are false positives (very flat areas that are not lakes) or are very small (representing seasonal puddles, not lakes). Skip these polygons.\\n   - Repair any areas where the polygon deviates from the true shoreline of the lake. Check for areas where artifacts in the DEM data may have caused sharp deviations in the shoreline, or where flat marshy areas were incorrectly classified as part of the lake.\\n   - If this lake is already mapped in OSM, select the new geometry and the existing lake, and run **Replace Geometry** (Cmd/Ctrl + Shift + G). This is important to preserve the ID and edit history of the element. If it did not exist in OSM, add the tags `natural=water` and `water=lake` to the new geometry. If the lake is large, it may have a name; consider checking USGS Topo maps.\\n   - Repair any nearby features that are affected by the changes to the lake geometry. In particular: waterways may need to be reconnected to the lake perimeter, and trails or landcover may incorrectly overlap the new geometry. Fix these before proceeding.\\n   - Do a final visual check to ensure that the edited lake\'s geometry is accurate. Use the backtick (`\\\\``) key to quickly cycle through all active imagery layers.\\n   - Consider deleting the generated lake polygon from the GeoJSON data layer as a way to track your progress.\\n6. Once you have examined every generated polygon in the GeoJSON data layer, upload your changeset. The suggested changeset comment should contain the tags #osmus-project-585 and #2024WRRHydrography - please keep these to identify your changeset as part of this effort.","perTaskInstructions":""}],"difficulty":"MODERATE","mappingPermission":"LEVEL","validationPermission":"LEVEL","enforceRandomTaskSelection":false,"private":false,"changesetComment":"-585","osmchaFilterId":null,"dueDate":"2024-07-01T07:00:00.000Z","imagery":"Bing","idPresets":[],"extraIdParams":null,"rapidPowerUser":false,"mappingTypes":["OTHER"],"campaigns":[],"organisation":15,"organisationName":"OpenStreetMap US","organisationSlug":"openstreetmap-us","organisationLogo":"https://www.openstreetmap.us/img/favicon/android-icon-192x192.png","countryTag":[],"licenseId":null,"allowedUsernames":[],"priorityAreas":null,"created":"2024-05-13T22:03:11.649685Z","lastUpdated":"2024-05-13T22:03:11.623627Z","author":"jake-low","activeMappers":0,"percentMapped":0,"percentValidated":0,"percentBadImagery":0,"taskCreationMode":"GRID","teams":[],"mappingEditors":["JOSM","CUSTOM"],"validationEditors":["JOSM","CUSTOM"],"interests":[]}' \
  --compressed
```

Given that this is my first task, it crossed my mind that maybe this is actually a permissions issue with my account or something. In any case, I'd really appreciate help solving this when you have a chance. Let me know if there's more information I can provide that would be useful.

**Comment by KatarzynaBaczewska-TomTom** [comment]
Hello,

I faced the same issues when I created a new project:
1. After adding a grid (in geojson format), name, and clicking ‘Create’, this message pops up: 
![USTMError1](https://github.com/osmus/tasking-manager/assets/128388518/454517d6-5f15-4102-b13d-ff4e2290a6fe)

2. Then the project with the previously added grid appears in the ‘Draft’ tab. When I fill in the required fields and try to save  changes this message appears: 
![USTMError2](https://github.com/osmus/tasking-manager/assets/128388518/98c7a4af-71df-4ec0-8b43-d03515e7bf68) 

So far I have not faced this type of issue during the creation of new projects, so I would appreciate it if you could check and fix these problems. If any further information is required please do not hesitate to contact me.

**Comment by jidanni** [comment]
Even trying to fill in my name and save it on
https://tasks.openstreetmap.us/settings ...
Upon typing a word, the Save button becomes enabled, but then doesn't save anything.

**Comment by quincylvania** [comment]
This is now fixed. My apologies for the delay, we've been stretched pretty thin over the summer.
